### DISCUSSION TITLE
Aqara door/window sensor: New vs old version

### DISCUSSION THREAD

**Discussion by bronger** [Koenkk/zigbee2mqtt#23696] [Devices]
Aqara offers two ZigBee variants of its door sensor.  The old version is called “MCCGQ11LM” and the new one is called “DW-S03D”. The latter is marketed with the “T1” on the package.

old version: https://www.aqara.com/en/product/door-and-window-sensor/specs/
new version: https://www.aqara.com/en/door-and-window-sensor-t1/door-and-window-sensor-t1-specs

zigbee2mqtt seems to consider them the same device: https://www.zigbee2mqtt.io/devices/MCCGQ12LM.html.  I do not think this is a mistake, therefore, I have not filed an issue.  Still, I bought the old version and it works fine with zigbee2mqtt, and now I consider buying the new version and I wanted to make sure that this does not have downsides (maybe even advantages?).

Why I consider the new version: According to https://www.reddit.com/r/Aqara/comments/16z0k65/t1_window_sensor_and_t1_water_leak_sensor_vs/, the new version supports a more modern ZigBee protocol version and OTA updates. This sounds nice, plus the current price tag is slightly smaller than for the old version.

Do you think that it has any disadvantages to use the new version with zigbee2mqtt? Do you even know of things that work better, e.g. OTA updates?

Thank you!

**Comment by vulpes2** [comment]
I'd recommend against getting the MCCGQ11LM, I've had one just stop working after a battery replacement, and now it leaves my network immediately or goes silent after joining no matter what I do. From what I can tell, Aqara devices are generally not fully complaint to the zigbee spec and can lead to unpredictable behaviors. If the physical size of the sensor is not important for you, I've had good experiences with the IKEA PARASOLL sensor, which has roughly the same width as the MCCGQ11LM while being thicker and longer to accommodate the AAA battery.
